.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,89 @@
+> Thank you for submitting your change to the project, we really appreciate your help. 
+> 
+> For us to be able to review and merge your changes as quickly as possible, please review our pull request process.
+>
+> _(DELETE THIS PARAGRAPH AFTER READING)_
+>
+
+> ### Pull Request Title
+>
+> Plese ensure that you have included a meaningful title describing your change. 
+>
+> For example...
+>
+> - Added 'spo orgassetslibrary remove' command solving #1042
```

Comment:
@garrytrinder , @waldekmastykarz  if the keywords `Closes, Fixes`  auto closes the issue, then should we not change our one liner message examples from 
`Added 'spo orgassetslibrary remove' command solving #1042`
to 
`Added 'spo orgassetslibrary remove' command. Fixes #1042`

Reply:
Good catch.

I would say that anything related to a new command or enhancement should use `Closes` and a bug fix should use ‘Fixes‘.

Based on that, your example, would therefore be...

 `Added 'spo orgassetslibrary remove' command. Closes #1042`

Happy to make the change.

Reply:
Agree. Lets see what will be @waldekmastykarz 's opinion.

Reply:
Good idea. Since we're at it, how about rephrasing `Added` to `Adds` so that it's in line with `Closes/Fixes`?

Reply:
Happy with that, I’ll get those pushed shortly 👍🏻

Reply:
Agree.

Reply:
I think that we should just consider using the `Closes` keyword for everything and drop `Fixes`, it makes it easier all round I think.

Consider `Fixes 'spo listitem add' command. Fixes #1297` and `Fixes 'spo listitem add' command. Closes #1297`. I would prefer the latter.

Our PR examples would then be as follows...

- Adds 'spo orgassetslibrary remove' command. Closes #1042
- Extends 'spo site list' command with support for returning deleted sites. Closes #1335
- Fixes 'spo listitem add' command. Closes #1297

Reply:
I've pushed the above change and also reflected the change in the Linked Issue section, removing the `Fixes` keyword in favour of using `Closes` keyword.